string ades_name
AdesData ades
---
bool success
